### PR TITLE
homepage section and subsection featured articles

### PR DESCRIPTION
**Pull request by Mitso**

### CODE REVIEW

File: babycenter/templates/core/tags/section_listing_homepage.html [hunk 1]
```diff
@@ -2,41 +2,63 @@
...
+	{% for sub_section in sub_sections %}
+		{% load_descendant_articles_for_section sub_section featured_in_homepage=True count=2 as featured_articles %}
 			{% if featured_articles %}
-			<div class="subsection-container">
-				<h3 class="section-list__title">
-					<a href="{% pageurl sub_section %}">{{sub_section.title}}</a>
-				</h3>
-				<ul class="article-list">
-					{% for article in featured_articles %}
+				<div class="subsection-container">
```
Comment (peterp) [inline babycenter/templates/core/tags/section_listing_homepage.html]: please remove this container

File: babycenter/templates/core/tags/section_listing_homepage.html [hunk 2]
```diff
@@ -2,41 +2,63 @@
...
+					<h3 class="section-list__title">
```
Comment (peterp) [inline babycenter/templates/core/tags/section_listing_homepage.html]: `section__title section__title--subsection`

**Comment by Mitso** [comment]
@peterp please see screenshots

bd-44

**Comment by peterp** [comment]
@Mitso I've left some comments in the PR.

File: babycenter/static/css/modules/_m-section-list.scss [hunk 3]
```diff
@@ -12,17 +12,17 @@
 	background: $orange;
 }
 
-.section-list__title {
+.section__title--subsection {
```
Comment (peterp) [inline babycenter/static/css/modules/_m-section-list.scss]: these styles are already covered by section__title; please remove the duplicates and only stipulate the variations

**Comment by Mitso** [comment]
@peterp please review

**Pull request merged**